web/packages/teleterm/src/ui/DocumentGatewayApp/AppGateway.tsx

```diff
@@ -72,27 +73,13 @@ export function AppGateway(props: {
           />
         </Validation>
       </Flex>
-      {cliCommandPreview && (
-        <CliCommand
-          cliCommand={cliCommandPreview}
-          isLoading={props.changePortAttempt.status === 'processing'}
```

Comment:
I don't think it is needed, this is a local call so this state change isn't even visible.